### ISSUE TITLE
CouchDB timeout on Mavericks

### ISSUE THREAD

**Issue by nroper** [hoodiehq-archive/hood.ie-website-old#41]
Hi,

I'm trying to get a Hoodie app running on Mavericks, but getting errors.

I installed Homebrew into ~/.homebrew and then ran:

$ brew install couchdb

Got errors, which I fixed by installing latest Xcode CLT and re-installing Erlang

Then ran brew install couched again and it worked fine.

Next I ran:

$ npm install -g hoodie-cli
$ hoodie new testapp

That worked OK.

Finally, cd to test app and then ran:

$ hoodie start

Version: 0.3.2 (node v0.10.16, npm 1.3.15, platform: darwin)

Initializing...
CouchDB started: http://127.0.0.1:6003
Waiting for CouchDB [---*--] FAILED 

ERR! Error: Timed out waiting for CouchDB. These logs may help:

Checked the couchdb.err file:

heart_beat_kill_pid = 50683
heart_beat_timeout = 11
init terminating in do_boot ()
heart: Sat Dec  7 22:47:46 2013: Erlang is crashing .. (waiting for crash dump file)
/Users/nickroper/.homebrew/Cellar/couchdb/1.5.0/bin/couchdb: line 336: echo: write error: Broken pipe
heart: Sat Dec  7 22:47:47 2013: Executed "/Users/nickroper/.homebrew/Cellar/couchdb/1.5.0/bin/couchdb -k" -> 256. Terminating.

I can start CouchDB on its own OK though:

$ /Users/nickroper/.homebrew/bin/couchdb
Apache CouchDB 1.5.0 (LogLevel=info) is starting.
Apache CouchDB has started. Time to relax.
[info] [<0.31.0>] Apache CouchDB has started on http://127.0.0.1:5984/

Any ideas?

Thanks

**Comment by janl** [comment]
Current hoodie expects Homebrew in /usr/local. There is a PR in the works that fixes this.

**Comment by nroper** [comment]
Fixed it...

I forgot to say that the first time I tried to run hoodie start I got a CouchDB timeout. I found another issue thread relating to this and the fix was to hard code couchdb_path and default_sys_ini - which I did.

However, I had only put the path for default_sys_ini and not included the default.ini file name.

Just fixed that and tried again and was prompted for admin password - test app is now up and running in the browser :-)

Nick
